install-tanzu-cli.md

```diff
@@ -407,7 +520,7 @@ Follow these instructions to update the Tanzu CLI that was installed for a previ
 3. Navigate to [Tanzu Application Platform](https://network.tanzu.vmware.com/products/tanzu-application-platform/)
 on Tanzu Network.
 
-4. Click the **tanzu-cli-v0.11.1** directory.
+4. Click the **tanzu-cli-tap-1.0.2** directory.
```

Comment:
Has the name of cli bundle changed from v<version> to tap-<version>? Also, in all the versions mentioned, we seem to prefix "v", but here it is removed. Is it intentional?

Reply:
changed directory back to tanzu-cli-v0.11.1.